src/gluonts/evaluation/_base.py

```diff
@@ -387,6 +437,40 @@ def smape(target, forecast):
         )
         return smape
 
+    @staticmethod
+    def owa(target, forecast, train_target, seasonal_error, freq, start_date):
```

Comment:
Why is the argument here called `train_target`?

Reply:
Because the naive predictor calls it target, but its the train sequences.

Reply:
There’s no training going on here. Is it the “past data” that precedes the target?

Reply:
yes

Reply:
ok so maybe you can change it to `past_target` I would say